src/BibleWell.App/App.axaml.cs

```diff
@@ -24,7 +29,7 @@ public override void OnFrameworkInitializationCompleted()
         var serviceProvider = services.BuildServiceProvider();
 
         // configure Avalonia app main window
-        var mainViewModel = serviceProvider.GetRequiredService<MainViewModel>();
+        var mainViewModel = serviceProvider.GetRequiredService<OpenFileViewModel>();
```

Comment:
Would expect to see this new view implemented as a separate 'page' (or view). However, I understand that finding a working example of switching views when not using the ReactiveUI seems to be almost impossible.

Reply:
yeah, I also wasn't sure if that would be too far out of scope of this. I'm a little trepidatious about researching and. adding a major piece of architecture when it's a poc for opening a file essentially.